engine/src/test/java/com/datatorrent/stram/codec/DefaultStatefulStreamCodecTest.java

```diff
@@ -214,4 +214,16 @@ public void testInnerClassSerialization() throws Exception
     }
   }
 
+  static class MyStreamCodec extends DefaultStatefulStreamCodec<String>
```

Comment:
can it be private? Can the name be changed to `StatefulStreamCodec` to avoid My...